bot.py

```diff
@@ -4,6 +4,119 @@
 import asyncio
 
 from secrets import token
+from random import randint
+from PIL import Image  # !Dependency! Pillow module is not standard
+
+# Since I don't know what I'm doing, I'm assuming I can write my definitions here and have them called within the client event
+def build(width=500, height=500):
```

Comment:
If you've already written this code elsewhere, lets add it to this project via a submodule. If you don't have it in another git repo, go ahead and take the code and put it into its own module and import it.

Reply:
Also, I've never used github for peer review on changes, so sorry If I'm doing weird stuff.